readme.markdown

```diff
@@ -144,6 +144,23 @@ By default, uncaught exceptions in your tests will not be intercepted, and will
 - In-process reporting with https://github.com/DavidAnson/tape-player
 - Describe blocks with https://github.com/mattriley/tape-describe
 
+# command-line flags
+
+While running tests, top-level configurations can be passed via the command line to specify desired behavior.
```

Comment:
if we're going to make this section, then `--require` and `--import` need to be documented also.

Reply:
okay, just to clarify, I rather see `--require` and `--ignore` in the code? Kindly let me know if there's more to it.

Reply:
oh right, sorry, yes, those are the two.

Reply:
updated pr to add documentation for `--require` and `--ignore` so far